frontend/src/views/camp/Material.vue

```diff
@@ -79,6 +85,67 @@ export default {
 
     this.camp().activities().$reload()
   },
+  methods: {
+    async downloadXlsx() {
+      await this.camp().activities().$loadItems()
+
+      var workbook = XLSX.utils.book_new()
+
+      var sheets = await Promise.all(
+        this.camp()
+          .periods()
+          .items.map(async (p) => {
+            var rows = await Promise.all(
+              p.materialItems().items.map(async (mi) => {
+                var activity = await this.getActivity(mi)
+                return [
```

Comment:
It looks like it work, but in my case it uses always the first activity name:
![image](https://user-images.githubusercontent.com/1506818/194712881-7e80e53d-0708-44cb-93ce-f45da23ebb74.png)



Excel output:
<!DOCTYPE html>

<html>
<head>
	
</head>

<body>


Quantity | Unit | Article | Material list | Activity
-- | -- | -- | -- | --
5 | Stk | Gruppenzelte | Pfadiheim |  
2 | Stk | Spatz-Zelte | Pfadiheim | Test
  |   | Blaues Knotenbüchlein | Bi Pi | Test
  |   | J+S Broschüren | Baghira | Test
  |   | Gitarre, Ipod und Böxli | Baghira | Test
1 |   | Papierrolle | Coop | Test
16 |   | Blätter für Regeln | Coop | Test
1 |   | ausgedruckte Liste mit Charakter | Castor | Test
2 |   | Karte mit eingezeichneter Route | Taifun | Test


</body>

</html>
[Materialliste(3).xlsx](https://github.com/ecamp/ecamp3/files/9739505/Materialliste.3.xlsx)

Reply:
@pmattmann geht das nicht, oder ist das ungewünscht?